api/src/modules/indicator-records/services/impact-calculator.service.ts

```diff
@@ -124,13 +125,12 @@ export class ImpactCalculator {
     if (!materialH3s) {
       throw new MissingH3DataError();
     }
-    const indicatorNameCodes: INDICATOR_TYPES_NEW[] =
-      Object.values(INDICATOR_TYPES_NEW);
+
     const indicatorsToCalculateImpactFor: Indicator[] = await getManager()
       .createQueryBuilder()
       .select()
       .from(Indicator, 'i')
-      .where('i.nameCode IN (:...nameCodes)', indicatorNameCodes)
+      .where('i.status = :status', { status: INDICATOR_STATUS.ACTIVE })
       .getMany();
```

Comment:
I would probably refactor this into a getActiveIndicators or something like that in the Indicator service/repository
The function in the service could construct a base options object as { status: active} and then merge the incoming options, and use those for a normal indicatorRepository.find()

Reply:
Totally agree, I left this at is was to have any dependencies and to be as much isolated as possible while we validate it, but I guess we should refactor it from here